### PR TITLE
parse_ini_file comment cleanup

### PR DESCRIPTION
**Pull request by Crell**
All of these updates are based on comments on the page, most of which I have since deleted as they'll either be incorporated here or we'll decide they just don't matter.

### CODE REVIEW

File: reference/filesystem/functions/parse-ini-file.xml [hunk 1]
```diff
@@ -32,7 +32,8 @@
      <term><parameter>filename</parameter></term>
      <listitem>
       <para>
-       The filename of the ini file being parsed.
+       The filename of the ini file being parsed. If a relative path is used,
+       it is evaluated relative to the current include path, then the current working directory.
```
Comment (cmb69) [inline reference/filesystem/functions/parse-ini-file.xml:36]: Is this true? [`include`](https://www.php.net/manual/en/function.include.php) is documented to work differently, and I'm not even sure if the `include_path` is used by `parse_ini_file` at all.

Comment (Crell) [reply reference/filesystem/functions/parse-ini-file.xml:36]: Frankly I'm not even sure.  That's what one of the comments said.  I've not tested it myself.

Comment (Crell) [reply reference/filesystem/functions/parse-ini-file.xml:36]: OK, just tested, and this isn't quite right.

* It *does* read from the include path.
* The cwd is read first.  If a file is in cwd and in the include path, the cwd file gets used.

I don't understand why it was written this way, but that's how it seems to work, so... :man_shrugging: 

I'll update the text momentarily.

**Comment by cmb69** [comment]
Thanks! Applied as http://svn.php.net/viewvc?view=revision&revision=350855.

**Pull request closed without merging**